apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V53__Tech_debt_db_cleanup.sql

```diff
@@ -0,0 +1,13 @@
+-- both psql and hsql support non-primary key index renaming
+ALTER INDEX beneficiary_monthly_year_month_partd_contract_bene_id_idx RENAME TO beneficiary_monthly_partd_contract_year_month_bene_id_idx;
+
+-- to check if an index is not used in postgres:
+-- select * from pg_stat_all_indexes where schemaname = 'public' and relname = 'beneficiary_monthly'
+DROP INDEX IF EXISTS beneficiary_monthly_partd_contract_number_year_month_idx;
+
+DROP TABLE IF EXISTS bene_hist_temp CASCADE;
+DROP TABLE IF EXISTS ccw_load_temp CASCADE;
```

Comment:
I think these are just cruft tables that were not created by flyway? I didn't think we were dropping these in here.

Also, I believe there is another unused table BeneficiariesHistory_old which is created by Flyway (V14 script) that should be dropped here though, right?

Reply:
So I didn't touch BeneficiariesHistory_old as I didn't know who created it; happy to nuke it though!

As for those other tables, you are right in that they weren't created by flyway, but we can certainly use flyway to remove them (espec with the DROP IF EXISTS). I know karl gets antsy about OOB db stuff so I added them to flyway cleanup.

Reply:
I think a comment is warranted to explain that these are unused tables that were created out of band then. Also, there are three more tables on prod that are in the same category.

Reply:
Also, I know I can be blind, but I just don't see the BeneficiariesHistory_old table..
![Screen Shot 2022-02-07 at 3 55 38 PM](https://user-images.githubusercontent.com/28897779/152870497-10373304-67c1-4838-90a5-9425a829cff6.png)

Reply:
I'll put the beneficiary_monthly_audit on the 'hit list' for later; it contains all the 2020,2021 bene monthly records that were probably erroneously reported back at the beginning of 2021. We alerted peering partners to this table if they needed to check which data BFD sent out between JAN and MAR of 2021 that had the wrong contract info due to the ref year faisco.

Reply:
Interesting that the BeneficiaryHistory_old is not there in PROD. It is there in sandbox and test though.

There are a few others ones in sandbox too